lib/utils/read_file.rb

```diff
@@ -0,0 +1,24 @@
+# encoding: utf-8
+# author: ERAMOTO Masaya
+
+module ReadFile
```

Comment:
Our other utilities are named as nouns... ObjectTraverser, CommentParser... while yours is named like a verb/action. Can I suggest changing this to `FileReader` instead (and alter the file name as well to `file_reader.rb`)?